src/test/java/com/wikia/webdriver/testsuites/testSuite.xml

```diff
@@ -45,6 +45,7 @@
             <class name="com.wikia.webdriver.testcases.adstests.TestParamValue"/>
             <class name="com.wikia.webdriver.testcases.adstests.TestProvidersChain"/>
             <class name="com.wikia.webdriver.testcases.adstests.TestRoadblocksAfterMultiplePageViews"/>
+            <class name="com.wikia.webdriver.testcases.adstests.TestSlotsOasis"/>
```

Comment:
Test runs 6 times for each slot in dataProvider (it means a browser has been opened and closed 6 times).
For such simple test I would rather check all slots at once.